src/Test/Rexl.Code.Test/Scripts/CodeGen/Functions/String.txt

```diff
@@ -71,3 +71,14 @@ Text.TrimEnd(Wrap(Null(""))) | With(_, {S: it, L: Text.Len(it)})
 Text.Replace(N, "A", "B")
 Text.Replace("A", N, "B")
 Text.Replace("ABC", Wrap("B"), N)
+
```

Comment:
Should also have:
* Tests for lifting.
* Tests of cases that aren't reduced during binding (once `ReduceCore` is implemented). That's the purpose of using `Wrap`.